### DISCUSSION TITLE
Font sizes do not match actual values

### DISCUSSION THREAD

**Discussion by jamal-ali1989** [johnfactotum/foliate#1416] [Q&A]
When I set font size to say 12, for example, even at 100% zoom, it is way smaller than actual fonts of size 12. Is there another setting missing? Or does the size represent something other than pt values?

**Comment by johnfactotum** [comment]
The value is in px, not pt. See https://github.com/johnfactotum/foliate/commit/10ab3a3ea7eeb8af8c4241683be3579c075e01c8. Also the book itself can contain styles that override the size.

**Comment by jamal-ali1989** [comment]
Thank you. Is px 1/96th of an inch or pixels in this scenario? 
I set the font to 72 and made it override publisher font. You can see the measurements with the pixel ruler. My screen is 1920*1080 pixels, DPI is 127, 5 pixels per millimeter.
![image](https://github.com/user-attachments/assets/c47b72b8-a0a3-44ff-99d2-5f21bd928e3d)

**Reply by johnfactotum** [reply]
Well, the default font size setting in Foliate is just the WebKit Settings property [`default-font-size`](https://webkitgtk.org/reference/webkit2gtk/stable/property.Settings.default-font-size.html). It's not specified in the documentations but I would expect the unit of this property to be the same as `px` in CSS. And one CSS `px` is indeed 1/96 CSS `in`, but a CSS `in` is not necessarily one physical inch.

Also note that "Override Publisher Font" does not currently force font size. It only forces the font family.

If you need precise control over the font size, you can do it with the [user stylesheet](https://github.com/johnfactotum/foliate/blob/gtk4/docs/faq.md#can-i-set-my-own-custom-css-styles) and set any CSS style you'd like.

**Comment by jamal-ali1989** [comment]
Thank you, then I guess this has to do with override publisher font option. Otherwise the fonts at 72 would be a little over twice as large.
